deploy_to_s3.py

```diff
@@ -0,0 +1,114 @@
+"""
+Module to deploy WhoTracksMe site to an s3 bucket.
+
+Usage:
+    deploy_to_s3 <bucket_name> [<prefix>] [--production]
+
+Options:
+    -h, --help                  Show help message.
+    --production                Production deployment (set cache-control metadata)
+"""
+import os
+import boto3
+from docopt import docopt
+from mimetypes import MimeTypes
+
+def list_full_bucket(s3_client, bucket_name, bucket_prefix):
```

Comment:
I don't know if I understood correctly, but if you are trying to list a bucket recursively, there is a `Pager` object that one can use:
```python
def iterate_bucket(bucket, prefix):
    client = boto3.client('s3')
    paginator = client.get_paginator('list_objects_v2')

    for page in paginator.paginate(Bucket=bucket, Prefix=prefix):
        for item in page['Contents']:
            yield bucket, item['Key']
```

Reply:
Interesting. This must've been hidden in the docs somewhere...